### DISCUSSION TITLE
Installed Chia GUI(.deb) but can't activate environment via CLI

### DISCUSSION THREAD

**Discussion by pureevans** [Chia-Network/chia-blockchain#4269] [Q&A]
Hello,

I installed the chia GUI with the experimental installer because I kept having issues with the CLI installation (window opened but kept being blank, no login showed).
Everything works find and I've already plotted a few TB, but now i want to install some additional software which needs the chia environment to be active (venv) in order to be installed. Based on the download side (https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian), if you install the GUI with the experimental installer, all CLI commands are in this directory (/usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/). I navigated there and tried . ./activate, but it keeps telling me bash...no file or directory found. 
Has anyone had the same issue and can help me?

EDIT: I managed to get it work by setting up my Linux Ubuntu again and installing the Chia Blockchain via Git Installation. I think this is the best way to get additional Software (e.g. Plotman) to work

**Comment by lvcivs** [comment]
I have not used the deb installer, so I can't confirm locally, but I suspect you may not even need the `. ./activate` command. 

What happens if you just use the chia file in that folder directly, for example by typing `chia version`? (It should just print the version number).

**Reply by pureevans** [reply]
My problem is that I need an active virtual environment (venv) in order to install my additional software, and if installed with deb I'm somehow not able to achieve that. I have also tried  . /chia to activate it, but no luck. I'm a software developer but only work with C# and windows, Linux is completely new to me. I'll try your answer out later, thanks for the reply.

**Reply by lvcivs** [reply]
I just tried this on a fresh VM install of Ubuntu and it is as I suspected. It will work like this:

1. `cd /usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/`
2. `./chia version`

It prints the version number.

Obviously you can substitute any other chia command like `./chia plots create` or whatever.

**Reply by ebeng** [reply]
that is where the folder is when installing with the .deb container/package
but then.... there is still no virtual enviroment you can start? 
or did you manage to get the plotman work now?

**Reply by lvcivs** [reply]
As I said, starting the `venv` is not needed if you use the .deb installer. I don't know about plotman, have never used it.

**Comment by 7by24** [comment]
.blank./activate ,not ../activate

**Reply by pureevans** [reply]
hey, thanks you, on my Linux machine I typed it with a blank space, since I'm on my phone it must have deleted the blank space between the dots in my post.

**Reply by 7by24** [reply]
I see, I recommend you try git instead of the installer, it's an easier way to install the software imo.
otherwise, you need to post the error logs here and we may see what actually happened.

**Reply by lvcivs** [reply]
The installation is definitely easier from the .deb, I mean that is literally why the installer exists in the first place. Also log files won't help if he can't even call the chia executable from the CLI.

**Reply by pureevans** [reply]
that's my plan. I tried the git installation first, but every time I opened the GUI all I saw was a blank window, with only the tabs on the upper left showing, but no content in the actual window (thus no ability to log into my wallet and do anything). 
I'm hoping that it will work this time, because I do prefer the "proper" installation way, but I couldn't get it to work for hours, that's why I turned to the experimental installer.

**Reply by pureevans** [reply]
That's my problem, it only says file or directory not found, that's it. I'll only get to it again in a few hours after work, I'll try git installation again and directly calling chia from CLI.

**Reply by 7by24** [reply]
> That's my problem, it only says file or directory not found, that's it. I'll only get to it again in a few hours after work, I'll try git installation again and directly calling chia from CLI.

i didn't install it via dpkg so I don't have much experience on this but I assume it caused by permission or environment variable issue and it's hard to be identified if you are not a Linux expert. that's why I suggested you try with git way on fresh installed OS

**Comment by ebeng** [comment]
I do have the same issue for the Plotman installation... 
did you figure it out?

**Reply by pureevans** [reply]
I'm also trying to install plotman, I'll report back as soon as I tried the two options mentioned above

**Comment by atyhieu** [comment]
Có

**Comment by GameJamer** [comment]
Reviving this post because I ran into same problem in version 1.2.11.
Installed from .deb file. CHIA command is not found, only CHIA-BLOCKCHAIN.

I did a workaround with creating an alias in BASH:

`alias chiaw="/usr/lib/chia-blockchain/resources/app.asar.unpacked/daemon/chia"`

Yet still... it would be nice to know .deb installer fix is on a way.

My version Linux is 5.10.0-10-amd64 #1 SMP Debian 5.10.84-1 (2021-12-08) x86_64 GNU/Linux.
